ext/gmp/tests/gh16411.phpt

```diff
@@ -0,0 +1,11 @@
+--TEST--
+GH-16411 (gmp_export() can cause overflow)
+--EXTENSIONS--
+gmp
+--FILE--
+<?php
+gmp_export(-9223372036854775808, 9223372036854775807, -9223372036854775808);
```

Comment:
Seems like we don't validate the `$option` argument properly?

Reply:
Ah, we're only checking for the relevant flags:

https://github.com/php/php-src/blob/79c71c9f0b5fa7e27d64075b624f92f07c64ad11/ext/gmp/gmp.c#L929-L958

So basically `$options % 32`. I think this okay (wouldn't want to touch it for a stable release), but could be improved for master (reject >= 32).

Reply:
Yeah, improving this for master is what I had in mind, I was just surprised that large int wasn't caught by the option checking.